hphp/hack/src/server/autocompleteService.ml

```diff
@@ -294,10 +294,10 @@ let autocomplete_member ~is_static env class_ cid id =
        more stable. *)
     if is_static || match_both_static_and_instance then (
       List.iter
-        (get_class_elt_types env class_ cid (Cls.smethods class_ |> sort))
+        (get_class_elt_types ~is_method:true env class_ cid (Cls.smethods class_ |> sort))
```

Comment:
Wondering if there's a better way to do this... given a value of type class_elt, can we retrieve the fact that it references a method or property? I wasn't sure how, which is why I'm threading that information in from here.

Reply:
It doesn't look like it. `class_elt` is defined here:

https://github.com/facebook/hhvm/blob/154e3b90833ff874ce8dcac9728754072134766e/hphp/hack/src/typing/typing_defs.ml#L86-L93

and flags only has things like 'is abstract':

https://github.com/facebook/hhvm/blob/154e3b90833ff874ce8dcac9728754072134766e/hphp/hack/src/typing/typing_defs_flags.ml#L34-L47

I suspect this is because we're trying to keep decls ('declarations', i.e. signatures) as small as possible because we have to fetch lots of them when type checking a file that depends on others. I agree the current code is pretty reasonable.